### PR TITLE
Update Dispatcher-Service.md

### PR DESCRIPTION
**Pull request by Cormoran96**
Add python3-pymysql

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Dispatcher-Service.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ available, but the testing package is working fine, you can grab it on
 package may be updated and have a new version number).
 
 ```
-apt install python3 python-mysqldb
+apt install python3 python-mysqldb python3-pymysql
```
Comment (SourceDoctor) [inline doc/Extensions/Dispatcher-Service.md:235]: when you change it like this, is python-mysqldb still needed?

Comment (Cormoran96) [reply doc/Extensions/Dispatcher-Service.md:235]: Yes it is. If you don't have you have an python error

Can not find mysqld module or something like that

**Comment by murrant** [comment]
@Cormoran96  python3-pymysql is installed by `pip3 install -r requirements.txt` (as well as other requirements) Did you run that?

**Comment by Cormoran96** [comment]
Ok I missed that information. I see now. I close my pull request

**Pull request closed without merging**